diff -aurN gtk-engines-2.6.3.orig/engines/clearlooks/Makefile.am gtk-engines-2.6.3/engines/clearlooks/Makefile.am
--- gtk-engines-2.6.3.orig/engines/clearlooks/Makefile.am	2005-04-12 17:55:07.000000000 +0200
+++ gtk-engines-2.6.3/engines/clearlooks/Makefile.am	2005-04-12 21:06:36.000000000 +0200
@@ -1,6 +1,6 @@
 INCLUDES = $(GTK_CFLAGS)
 
-enginedir = $(libdir)/gtk-2.0/$(GTK2_VERSION)/engines
+enginedir = $(libdir)/gtk-2.0/$(GTK_VERSION)/engines
 engine_LTLIBRARIES = libclearlooks.la
 
 libclearlooks_la_SOURCES =		\
